modules/starfish/2.0/config/default.yaml

```diff
@@ -0,0 +1,51 @@
+lcr-modules:
+    
+    starfish:
+
+        dirs: 
+            
+            _parent: "__UPDATE__"
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            # Each key:value pair is the name of the variant caller and the path to its VCF file
+            # Each key should be unique in this set. 
+            vcf: 
+                strelka: "__UPDATE__"
+                
+
+            starfish_script: "{MODSDIR}/src/starfish.py"
+
+        scratch_subdirectories: []
+
+        options:
+            starfish_run: "--sample ALT --squash-ploidy --vennout-format pdf"
+            starfish_union: "-m both"
+
+        conda_envs:
+            bcftools: "{MODSDIR}/envs/bcftools-1.10.2.yaml"
+            starfish: "{MODSDIR}/envs/starfish.yaml"
+            
+        threads:
+            starfish_run: 4
+            starfish_union: 4
+
+        resources:
+            starfish_run: 
+                mem_mb: 15000
```

Comment:
It was for 90% of the files, but for the others I had to up it to 100000. I'm wary of making such a high number the default because it can slow its progression on the cluster. But maybe I should add the attempt feature here.

Reply:
I 100% think the attempt feature should be used here. This is what got me thinking last night about the current conflict between resource unpacking and that feature.

Reply:
I am going to add the attempt feature in a future version bump when we can implement it more widely.